servers/src/main/java/tachyon/worker/block/BlockLockManager.java

```diff
@@ -120,9 +118,9 @@ public void unlockBlock(long lockId) throws NotFoundException {
     Lock lock;
     synchronized (mSharedMapsLock) {
       LockRecord record = mLockIdToRecordMap.get(lockId);
-      if (null == record) {
-        throw new NotFoundException("Failed to unlockBlock: lockId " + lockId
-            + " has no lock record");
+      if (record == null) {
```

Comment:
Yeah, I think that's a habit learned from C, but I don't think it's necessary in Java, since Java requires that the expression in an if evaluate to a boolean value, so unless the constant is a boolean, you'd get a compilation error either way you put the arguments. (and if it is a boolean, you shouldn't be using == anyway...). Right?